src/commands/upload_file.ts

```diff
@@ -32,20 +35,40 @@ interface UploadFileParameter {
 	driveKey?: DriveKey;
 }
 
+type FilePath = string;
+
+const formatResults = (results: ArFSResult[]): ArFSResult => {
+	const isFolderUploadResults = results.length === 1;
+	if (isFolderUploadResults) {
+		return results[0];
+	}
+	return results.reduce(
+		(previousValue, currentValue) => ({
+			created: [...previousValue.created, ...currentValue.created],
+			tips: [...previousValue.tips, ...currentValue.tips],
+			fees: { ...previousValue.fees, ...currentValue.fees }
+		}),
+		{ created: [], tips: [], fees: {} }
+	);
+};
+
 new CLICommand({
 	name: 'upload-file',
 	parameters: [
-		ParentFolderIdParameter,
-		LocalFilePathParameter,
-		DestinationFileNameParameter,
-		LocalFilesParameter,
 		BoostParameter,
+		DestinationFileNameParameter,
 		DryRunParameter,
+		GlobParameter,
+		LocalFilePathParameter,
+		LocalFilesParameter,
+		ParentFolderIdParameter,
 		...ConflictResolutionParams,
 		...DrivePrivacyParameters
 	],
 	action: new CLIAction(async function action(options) {
 		const parameters = new ParametersHelper(options);
+		const globValue = parameters.getParameterValue(GlobParameter);
```

Comment:
Nit: could improve the locality of reference by moving this to where it's used down below.

Reply:
Great pick... I let it there because at first I was using this const to change the results report.